server/crawler/package_repository_metadata_test.go

```diff
@@ -31,9 +31,61 @@ func TestGetLocator_Success(t *testing.T) {
 		time.Time{},
 		"",
 	)
-	
+
 	expectedLocator = "kurtosis-tech/awesome-kurtosis/flink-kafka-example/kurtosis-package/"
 
 	require.Equal(t, expectedLocator, packageRepositoryMetadata.GetLocator())
 
 }
+
+func TestGetDownloadRootURL_Success(t *testing.T) {
```

Comment:
@adschwartz adding the pending unit test you suggested here: https://github.com/kurtosis-tech/kurtosis-package-indexer/pull/72#discussion_r1414403660